policies/queries/ec2/nat_gateways_unused.sql

```diff
@@ -0,0 +1,14 @@
+insert into aws_policy_results
+with address as (select distinct nat_gateway_cq_id from aws_ec2_nat_gateway_addresses)
+select :'execution_time'                       as execution_time,
+       :'framework'                            as framework,
+       :'check_id'                             as check_id,
+       'Unused NAT gateways should be removed' as title,
+       gateway.account_id,
+       gateway.id                              as resource_id,
+       case
+           when address.nat_gateway_cq_id is null then 'fail'
+           else 'pass'
+           end                                 as status
+from aws_ec2_nat_gateways gateway
+         left join address on address.nat_gateway_cq_id = gateway.cq_id
```

Comment:
Is it possible to create a NAT Gateway and not attach it to a VPC?

Reply:
good catch, removing